### PR TITLE
Reorg, add Gitlab & Gitlab CI

### PR DESCRIPTION
**Pull request by ksdme**
As of now we are still public badges of repos for getting their build statuses. Example: https://gitlab.com/coala/mobans/badges/master/pipeline.svg

### CODE REVIEW

**Comment by ksdme** [comment]
@jayvdb PTAL.

File: README.md [hunk 1]
```diff
@@ -1,36 +1,108 @@
...
-Organization has to be in scope of the access token. If it is a public org then `repo` scope should be sufficient. Use `--no-color` for no colored mode.
+Git host specific tokens are required to be available as environment variables. Org name should be in the form of `host:org`, if no host is passed then it defaults to `github`. Organization has to be in scope of access token. Use `--no-color` for no colored mode.
 
-## Result
+Example:
 ```shell
-https://github.com/coala/coala-vs-code: passing
-https://github.com/coala/coala-quickstart: failing
-https://github.com/coala/rultor-python: failing
+(env) ksdme@gridComputer:~/Work/coala/org-status$ GITHUB_TOKEN=6729fb81ef73cad643f5f0729c14551788e2a084 python org-status.py github:coala --verbose
```
Comment (jayvdb) [inline README.md]: needs to be a fake token of course.

File: org-status.py [hunk 2]
```diff
@@ -1,74 +1,86 @@
...
+    return environ['{}_TOKEN'.format(host_name.upper())]
 
-def get_travis_status(repo, branch='master'):
-  url = 'https://api.travis-ci.org/coala/{repo}.svg?branch={branch}'.format(
-          repo=repo, branch=branch)
 
-  svg_result = requests.get(url)
-  svg = svg_result.text
+def get_org_host(host_org):
+    parts = host_org.strip().split(':')
```
Comment (jayvdb) [inline org-status.py]: `.split(':', 1)`

or better
```py
host, _, org = host_org.partition(':')
```

File: org-status.py [hunk 3]
```diff
@@ -1,74 +1,86 @@
...
 
-  return process_travis_result(svg)
+    # defaults to gh
+    if len(parts) == 1:
+        parts = ['github', parts[0]]
```
Comment (jayvdb) [inline org-status.py]: more useful if it doesnt default to `github` , but instead does multiple hosts.

so we can run it once for coala and see results both gitlab and github.

e.g. `org-status coala` , or also good `org-status :coala`

the colon is only needed to reliably parse orgs who are also hosters

- http://gitlab.com/github and https://gitlab.com/travis-ci are squatted names.
- http://github.com/gitlab and https://github.com/gitlab-ci are real
- https://github.com/travis-ci is real

File: org-status.py [hunk 4]
```diff
@@ -1,74 +1,86 @@
...
+
+    return parser
+
+
+def main():
+    parser = get_argument_parser()
+    args = parser.parse_args()
+
+    Host, org = get_org_host(args.org)
+    token = get_host_token(Host.HostName)
```
Comment (jayvdb) [inline org-status.py]: would be good to catch `KeyError: 'GITHUB_TOKEN'`

and give slightly user-friendly error

File: README.md [hunk 5]
```diff
@@ -1,36 +1,108 @@
...
-https://github.com/coala/coala-eclipse: failing
+https://github.com/coala/gh-board: failing
 https://github.com/coala/PyPrint: passing
-https://github.com/coala/coala: failing
-https://github.com/coala/coala-emacs: failing
 https://github.com/coala/documentation: failing
-https://github.com/coala/corobo: passing
+https://github.com/coala/coala-emacs: failing
+https://github.com/coala/coala: passing
+https://github.com/coala/docker-coala-base: error
```
Comment (jayvdb) [inline README.md]: I also get an error for https://github.com/coala/docker-coala-base , but `--verbose` doesnt say what happened.  can it show the problem?

File: README.md (hunk 1 above)
Comment (ksdme) [reply README.md]: LOL! Revoked and updated.

**Pull request merged**